ga4gh/backend.py

```diff
@@ -26,6 +27,19 @@ def __init__(self, dataDir, variantSetClass):
                 self._variantSetIdMap[variantSetId] = variantSetClass(
                     variantSetId, relativePath)
         self._variantSetIds = sorted(self._variantSetIdMap.keys())
+        self._nextReferenceSetId = 0
```

Comment:
http://ga4gh.org/documentation/api/v0.5.1/ga4gh_api.html#/schema/org.ga4gh.GAReferenceSet defines GAReferenceSet.id as a unique string; you're using sequential integers?